sig-advocacy-and-outreach/summer-ospp/README.md

```diff
@@ -0,0 +1,23 @@
+KubeSphere in [Summer 2021 of Open Source Promotion Plan](https://github.com/summer-ospp). You can find our proposals in this page. Or you also can submit your proposal to KubeSphere community.
+
+## Proposals
+
+| Project | Area | Skills to study/improve |
+| --- | --- | --- |
+| [KubeSphere Jenkins Client Refactor](kubeSphere-jenkins-client-refactor.md) <br/>Split the Jenkins client from KubeSphere. Or use an existing Jenkins client. </br>Potential Mentor(s): [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/)| DevOps | Golang, REST API/OpenAPI, Jenkins, Kubernetes |
+| [KubeSphere Tekton Integration](kubeSphere-tekton-integration.md) <br/>Integrate Tekton as an alternative CI/CD engine of KubeSphere DevOps. <br/>Potential Mentor(s): [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/)| DevOps | Golang, Tekton, Jenkins, Kubernetes |
+| [Customize Notification Receivers Using a Webhook](customize-notification-receivers-using-a-webhook.md) <br/>Enable [Notification Manager](https://github.com/kubesphere/notification-manager) to obtain a notification receiver list from a webhook so that users can customize the notification receivers. <br/>Potential Mentor(s): [Benjamin Huo](https://github.com/benjaminhuo), [Wanjun Lei](https://github.com/wanjunlei) | Notification Manager | Golang, REST API, Kubernetes |
+| [Expose LoadBalancer Services Using iptables NAT Rules](expose-loadbalancer-services-using-iptables-nat-rules.md) <br/>Provide a simple mode of [PorterLB](https://porterlb.io/), which allows users to use the IP address of a Kubernetes cluster node and a service port to expose a LoadBalancer service. <br/>Potential Mentor(s): [Zhengyi Lai](https://github.com/zheng1), [Jiong Duan](https://github.com/duanjiong) | Network | Golang, Kubernetes, Linux network (iptables and IPVS) |
+
+## Students
+
+* [Students application process](students-application-process.md)
+* [Open Source Promotion Plan Portal](https://summer.iscas.ac.cn/))
+
+## Mentors
+
+* [Summer 2021 of Open Source Promotion Plan Portal](https://summer.iscas.ac.cn)
+
+## Contacts
+
+* TBD
```

Comment:
What kind of Contact should we leave here? Email, mailing list, or WeChat group?

Reply:
I suggest WeChat group.